### ISSUE TITLE
Tab should not focus inside collapsed split view

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52596]
**Environment Details:**
VSCode Version : 1.24.0
OS Version : Win10

**Additional Details:**
MAS Violated: MAS 2.4.3 , MAS 1.3.1
ScreenReader Versions : NVDA2018.1.1 / JAWS18.0.5038

**Repro Steps:**
1. Launch VS Code.
2. Navigate to Activity Bar and select "Explorer"(Cntrl+Shift+E) button.
3. Navigate to "File Explorer" and "Open Editors" toolbars when collapsed. 
4. "Tab" after "Open Editors" toolbar (when collapsed).

**Actual:**
Keyboard focus is lost when "Tab" is pressed after "Open Editors" toolbar (when collapsed). Focus moves to treeview. Screenreader announces "treeview" and then on next "Tab" focus moves to next toolbar.

**Expected:**
Keyboard focus should move to the next toolbar when "Tab"  is pressed after "Open Editors" toolbar (when collapsed).

Screenreader should not read "treeview" and should read the next toolbar.

**Recommendations:**
Remove tab-index from the treeview when "Open Editors" toolbar is in collapsed state and "tab" is pressed.
or,
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
The keyboard only users will navigate to treeview which is not actionable item as well as is not present on the screen.

**MAS Reference:**
MAS 1.3.1 - https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

MAS 2.4.3 - https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9}

**Attachment for Reference:**
[A11y_VSCode_ViewSearch_Keyboard_KFLossWhenToolbarsCollapsed.pptx](https://github.com/Microsoft/vscode/files/2123450/A11y_VSCode_ViewSearch_Keyboard_KFLossWhenToolbarsCollapsed.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
I can repro this only on windows, not mac.
The issue here is that the collapsed split view should not focus inside.
Forwarding to @sandy081

**Comment by sandy081** [comment]
@isidorn This is nothing to do with split view. Each view has to hide its widget (tree or list) when it is not shown. Tree panel is doing correctly and the OpenEditorsView is not hiding the list. Fixed it.

**Comment by isidorn** [comment]
@sandy081 thanks for fixing this.
